castle/cms/subscribers.py

```diff
@@ -209,3 +217,111 @@ def on_youtube_video_state_changed(obj, event):
             tasks.youtube_video_state_changed.delay(obj)
         except CannotGetPortalError:
             pass
+
+
+def create_version_on_save(context, event):
```

Comment:
was this pulled/modified from elsewhere? if it was, it'd be good to have a reference to the original in a comment